src/main/java/org/folio/service/InvoiceLineStorageService.java

```diff
@@ -0,0 +1,84 @@
+package org.folio.service;
+
+import static org.folio.rest.impl.InvoiceStorageImpl.INVOICE_LINES_PREFIX;
+import static org.folio.rest.utils.ResponseUtils.buildErrorResponse;
+import static org.folio.rest.utils.ResponseUtils.buildNoContentResponse;
+import static org.folio.rest.utils.ResponseUtils.buildResponseWithLocation;
+import static org.folio.rest.utils.RestConstants.OKAPI_URL;
+
+import javax.ws.rs.core.Response;
+import java.util.Map;
+
+import org.folio.dao.lines.InvoiceLinesDAO;
+import org.folio.rest.jaxrs.model.InvoiceLine;
+import org.folio.rest.jaxrs.model.InvoiceLineAuditEvent;
+import org.folio.rest.persist.DBClient;
+import org.folio.service.audit.AuditOutboxService;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Handler;
+import io.vertx.ext.web.handler.HttpException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+
+@Log4j2
+@RequiredArgsConstructor
+public class InvoiceLineStorageService {
+
+  private final InvoiceLinesDAO invoiceLinesDAO;
+  private final AuditOutboxService auditOutboxService;
+
+  public void createInvoiceLine(InvoiceLine invoiceLine, Handler<AsyncResult<Response>> asyncResultHandler,
+                                Context vertxContext, Map<String, String> headers) {
+    try {
+      vertxContext.runOnContext(v -> {
```

Comment:
I think this declarations vertxContext.runOnContext is not needed, we do not have any blocking operations, calls to DB is anon blocking in our case because we are using asynchronus vertx PostgresClient

Reply:
Removed from all new post/put methods